src/languageProvider/locales/zh-Hans.json

```diff
@@ -157,10 +152,11 @@
   "walletUnlockDialog.unlockWallet": "解锁钱包",
   "walletUnlockDialog.walletPassphrase": "钱包密码",
   "walletUnlockDialog.walletPassphraseRequired": "这个操作需要解锁您的钱包，请输入您的密码.",
+  "withdrawDetail.alreadyWithdrawn": "You have already withdrawn with this address.",
   "withdrawDetail.returnRate": "回报率：",
-  "withdrawDetail.reward": "回报",
-  "withdrawDetail.winningOutcome": "胜出选项",
-  "withdrawDetail.withdrawTo": "提现到",
+  "withdrawDetail.reward": "REWARD",
```

Comment:
???

Reply:
Hm i didnt change these.. I just ran the script.